src/status.h

```diff
@@ -57,13 +57,18 @@ class Status {
     NetSendErr,
   };
 
-  Status() : Status(cOK, "ok") {}
-  explicit Status(Code code, std::string msg = "") : code_(code), msg_(std::move(msg)) {}
+  Status() : Status(cOK) {}
+  explicit Status(Code code, std::string msg = {}) : code_(code), msg_(std::move(msg)) {}
   bool IsOK() { return code_ == cOK; }
   bool IsNotFound() { return code_ == NotFound; }
   bool IsImorting() { return code_ == SlotImport; }
-  std::string Msg() { return msg_; }
-  static Status OK() { return Status(cOK, "ok"); }
+  std::string Msg() {
+    if (IsOK()) {
+      return "ok";
+    }
+    return msg_;
```

Comment:
Will this change cause branch prediction overhead and affect efficiency which we'd like to improve in this PR? It seems not a net win.

Reply:
The function `Msg` only will be called in a error path, so the probability of calling is low. This fits perfectly with my idea.